src/applications/bmqbrkr/bmqbrkr.m.cpp

```diff
@@ -672,6 +651,46 @@ int main(int argc, const char* argv[])
         return mqbu::ExitCode::e_COMMAND_LINE;  // RETURN
     }
 
+    if (version) {
+        bsl::cout << bmqbrkrscm::Version::version() << "\n";
```

Comment:
May be add a string like "BlazingMQ broker version: " before printing the version. Unless, you just want the broker to print the version so that some scripts can rely on the output?

Reply:
Anyway the more verbose output is not difficult to parse. Adding the string.

Reply:
If the main goal is to let some script invoke `/path/to/bmqbrkr -v` to figure out the version programmatically, then perhaps we should print only the verison w/o any string.  Will leave it up to you to make that call.